up42/catalog.py

```diff
@@ -386,6 +387,7 @@ def construct_order_parameters(
             image_id: The id of the desired image (from search results)
             aoi: The geometry of the order, one of dict, Feature, FeatureCollection,
                 list, GeoDataFrame, Polygon. Optional for "full-image products".
+            tags: Optional [List[srt]] Add tags to order with the provided tags.
```

Comment:
```suggestion
            tags: List of tags to be added to the order.
```
nitpick: consider removing `Optional [List[srt]]` from the doc-string to be consistent with other params. OR, we should add it to all other params as well to be consistent :)